test/helpers/mapboxMock.ts

```diff
@@ -0,0 +1,95 @@
+/* Mapbox GL mock helper for Vitest component tests.
+   Usage: import * as mapboxMock from '@/test/helpers/mapboxMock'
+   The first import registers the vi.mock('mapbox-gl', …) stubs. */
+
+import { vi } from "vitest";
+
+// internal state ---------------------------------------------------------
```

Comment:
```suggestion
// Internal state ---------------------------------------------------------
```

And similar capitalization convention across the file (sorry for the nit).